RussianFilter/sections/adservers.txt

```diff
@@ -5,6 +5,12 @@
 ! `window.adswellBaseDomain`
 ! https://damskiy.xyz/vendor/lib/oY.js
 !
+||xokikdassjjhrn.com^$document,popup
+||tmnlsmvgrkoj.com^$document,popup
+||autoizers.info^$document,popup
```

Comment:
Just remove it, you've already had `||autoizers.info^$third-party`.

Reply:
Also for the others?

Reply:
Can I use `$third-party` for all the domains I added and removing the `$document,popup`? Since it's not supported by the apps.

Reply:
Yes, for others too. After that I think there is nothing to do - looks ok.

Reply:
Ok done.